### PR TITLE
adding text utilities component

### PR DESCRIPTION
**Pull request by bipin7yadav**
-Headings
-small texts
-large texts

### CODE REVIEW

File: text-utility.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            <h3><a href="/rating comp/rating.html" class="side-nav">Rating</a></h3>
+            <h3><a href="/slider comp/slider.html" class="side-nav">Slider</a></h3>
+            <h3><a href="/text utilities/text-utility.html" class="side-nav">Text Utilities</a></h3>
+            
+
+        </div>
+
+
+        
+
```
Comment (mimansha-swarup) [inline text-utility.html]: remove unnecessary spaces

File: text.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+.headings{
+    background-color: var(--aqua-color);
+    font-size: large;
+}
+
+.centered{
+    text-align: center;
+    background-color: var(--orange-color);
```
Comment (mimansha-swarup) [inline text.css:83]: suggestion: give it align-self:center too
cause inside flex text-align : center will not work

**Review by someshtiwarii — APPROVED** [review]

**Review by mimansha-swarup — COMMENTED** [review]
few suggestions

File: text.css (hunk 2 above)
Comment (bipin7yadav) [reply text.css:83]: ok

File: text-utility.html (hunk 1 above)
Comment (bipin7yadav) [reply text-utility.html]: removed

**Review by manojkumar8400 — APPROVED** [review]

**Pull request merged**